apps/src/gamelab/GameLabP5.js

```diff
@@ -427,6 +428,22 @@ GameLabP5.prototype.init = function (options) {
     array.displace = _groupCollideGameLab.bind(array, 'displace');
     array.bounce = _groupCollideGameLab.bind(array, 'bounce');
 
+    /**
+     * Test each member of group against the target using the isTouching sprite
+     * method.  Return true if any touching occurred.
+     *
+     * @method isTouching
+     * @param {Object} target Group or Sprite
+     * @return {boolean} True if any touching occurred
+     */
+    array.isTouching = function (target) {
+      var didTouch = false;
+      for (var i = 0; i < this.size(); i++) {
+        didTouch = this.get(i).isTouching(target) || didTouch;
```

Comment:
Also, I was going to suggest an early-out here, but then I realized `isTouching()` has side effects on the sprites.  It might be worth a comment to that effect.